manuscript/book.md

```diff
@@ -454,7 +442,7 @@ Here we’re wrapping a single item in an array, so we can use the same code to
 
 ### Early return
 
-_Guard clauses_, or _early return_, is a great way to avoid nested conditions. Nested conditions, also known as the [arrow anti pattern](http://wiki.c2.com/?ArrowAntiPattern) and _dangerously deep nesting_, are often used for error handing:
+Applying _guard clauses_, or _early returns_, is a great way to avoid nested conditions. A series of nested conditions, also known as the [arrow anti pattern](http://wiki.c2.com/?ArrowAntiPattern) or _dangerously deep nesting_, is often used for error handing:
```

Comment:
Maybe
 
```suggestion
Using _guard clauses_, or _early returns_, is a great way to avoid nested conditions. A series of nested conditions, also known as the [arrow anti pattern](http://wiki.c2.com/?ArrowAntiPattern) or _dangerously deep nesting_, is often used for error handing:
```